scss/titles.scss

```diff
@@ -4,7 +4,7 @@ $text-big: 24px;
 
 .title {
   font-size: 24px;
-    padding: 32px 0 32px 50px;
+    padding: 65px 20px 20px 98px;
```

Comment:
hmmmm I think some of this can be done using margins

Reply:
with margins is harder to allign the title over the triangle image